shared/src/native-src/loader.cpp

```diff
@@ -37,8 +38,11 @@ namespace shared
         WStr("Datadog.AutoInstrumentation.Profiler.Managed"),
     };
 
-    constexpr const WCHAR* SpecificTypeToInjectName = WStr("System.AppDomain");
-    constexpr const WCHAR* SpecificMethodToInjectName = WStr("IsCompatibilitySwitchSet");
+    const WCHAR* SpecificTypeToInjectName = WStr("System.AppDomain");
+    const size_t SpecificTypeToInjectNameSize = WStrLen(SpecificTypeToInjectName);
```

Comment:
The GetMemberProps and GetTypeDefProps APIS are returning "string size"; i.e. final \0 included so add +1 to both SpecificTypeToInjectNameSize and SpecificMethodToInjectNameSize

To avoid confusion, I would also rename functionNameLength to functionNameSize and typeNameLength to typeNameSize